docs/cli/geo/search.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Search
+description: Use Amplify CLI to create and manage location search indices(or place indices) that are used to search for places in your application.
+---
+
+Amplify CLI's `geo` category enables you to create and manage place index resources used to search by places, addresses and coordinates in your application.
+You need to install Amplify CLI with the `@geo` tag in order to get the Geo functionality. You can use the following command to install this version globally.
+```console
+npm i -g @aws-amplify/cli@geo
+```
+
+## Setup a new Search Index
+
+You can add a new location search index by running the following command from your project's root folder:
+
+```bash
+amplify add geo
+```
+```console
+? Select which capability you want to add:
+  Map (visualize the GeoSpatial data)
+> Location search (search by places, addresses, coordinates)
+```
+If you haven't set up the `auth` category already, the Amplify CLI will guide you through a workflow to enable the auth category.
+
+```console
+? You need to add auth (Amazon Cognito) to your project in order to add geo resources. Do you want to add auth now?
+> Yes
+
+Using service: Cognito, provided by: awscloudformation
+```
+
+Next, you can set a friendly name for the location search index:
+
+```console
+? Provide a name for the location search index (place index):
+> MyPlaceIndex
+```
+
+### Search Access permissions
+
+Next, configure the access permissions for your location search index. You can scope permissions based on an individual user's authentication status.
+
+```console
+? Who can access this Search Index?
+❯ Authorized users only
+  Authorized and Guest users
+```
+
+Select `Authorized users only` if only authenticated users can have read-only permissions for the location search index.
```

Comment:
Same feedback as on maps, instead of "read-only" permission, tell me what the permission does. Can I search? Get results?